source/features/select-notifications.tsx

```diff
@@ -13,22 +13,27 @@ import {
 	GitPullRequestDraftIcon,
 	GitPullRequestIcon,
 	IssueOpenedIcon,
+	SquirrelIcon,
 	XCircleIcon,
 } from '@primer/octicons-react';
 
 import features from '../feature-manager.js';
 import observe from '../helpers/selector-observer.js';
 
+const prIcons = ':is(.octicon-git-pull-request, .octicon-git-pull-request-closed, .octicon-git-pull-request-draft, .octicon-git-merge)';
+const issueIcons = ':is(.octicon-issue-opened, .octicon-issue-closed, .octicon-skip)';
```

Comment:
Also more comprehensive bugfix for:

- https://github.com/refined-github/refined-github/issues/6801
- https://github.com/refined-github/refined-github/pull/6803